source/security/private-endpoints.txt

```diff
@@ -372,6 +372,15 @@ You can toggle this setting in three ways:
   <operation/allNonVpcClientRequests>` with a boolean value (``true`` to remove
   the default restriction, or ``false`` to reinstate it.)
 
+IP Access Lists and Private Endpoints
```

Comment:
If I'm reading this right - that Private Endpoints ignore the IP Access List - it feels a bit like we're burying the lede here. I might reframe/rename this section to something like:

"Private Endpoints Ignore IP Access Lists" and lead with that detail. Then, you could follow with the information that public requests still adhere to the IP Access List as expected.

Reply:
That's definitely clearer. Thank you!